programs/castle-vault/src/instructions/init_vault.rs

```diff
@@ -157,6 +157,7 @@ pub fn handler(
         last_update: LastUpdate::new(clock.slot),
     };
     vault.config = VaultConfig::new(config)?;
+    vault.lp_token_supply = 0;
```

Comment:
The code changes themselves are fine. But do we have a migration plan for existing vaults? Or are we going to go ahead with a new vault?

Reply:
Undecided whether we deploy a new vault (leaning towards yes). But if we don't, at deployment time we can just add a new IX that sets the right value.